dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/arquillian/arquillianexamples/WebappArquillianDeployment.java

```diff
@@ -64,7 +66,8 @@ public void after() throws Throwable {
         JavaArchive javaArchive = ShrinkWrap.create(JavaArchive.class, "dv-webapp.jar")
                 .merge(PersistenceArquillianDeployment.createDeployment())
                 .addAsManifestResource(new FileAsset(new File("src/main/webapp/WEB-INF/beans.xml")), "beans.xml")
-                .addPackages(true, "edu.harvard.iq.dataverse");
+                .addPackages(true, "edu.harvard.iq.dataverse")
+                .deleteClass(WorkflowExecutionWorker.class);
```

Comment:
Hmm maybe some kind of javadoc why this class was deleted?

Reply:
It is a leftover of my attempt to run integration tests in `dataverse-webapp` project. I will revert it as it should be done in separate task.